lib/containers/pr-review-comments-container.js

```diff
@@ -0,0 +1,139 @@
+import {graphql, createPaginationContainer} from 'react-relay';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {PAGE_SIZE, PAGINATION_WAIT_TIME_MS} from '../helpers';
+
+export class BarePullRequestReviewCommentsContainer extends React.Component {
+
+  static propTypes = {
+    collectComments: PropTypes.func.isRequired,
+    review: PropTypes.shape({
+      id: PropTypes.string.isRequired,
+      submittedAt: PropTypes.string.isRequired,
+      comments: PropTypes.object.isRequired,
+    }),
+    relay: PropTypes.shape({
+      hasMore: PropTypes.func.isRequired,
+      loadMore: PropTypes.func.isRequired,
+      isLoading: PropTypes.func.isRequired,
+    }).isRequired,
+  }
+
+  componentDidMount() {
+    this.handleComments();
```

Comment:
we are also calling this method in `_loadMoreComments` - it's the callback that gets passed to `relay.loadMore`.  We do want to keep calling `handleComments` each time we receive new props, because the parent component needs to know when all the comments have been received, so it can group them by `replyTo.id`.  Does the suggestion of making this a pure component still make sense under those conditions?